test/concept-coach/launcher/items.spec.coffee

```diff
@@ -11,8 +11,3 @@ describe 'Launcher SVG items', ->
     Testing.renderComponent( LaptopAndMug, props: @props ).then ({dom}) ->
       expect(dom.tagName).equal('svg')
       expect(dom.getAttribute('height')).equal('42px')
-
-  it 'renders background and desk without errors', ->
-    Testing.renderComponent( BackgroundAndDesk, props: @props ).then ({dom}) ->
-      expect(dom.tagName).equal('svg')
```

Comment:
Because the element was removed.  I guess I could technically change it to test for a div instead.